src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/MiscFeatures.kt

```diff
@@ -149,7 +150,7 @@ class MiscFeatures {
                 }
             }
             if (unformatted == "Your zapper is temporarily fatigued!") {
-                val duration = Duration.milliseconds(blockZapperCooldownExpiration - System.currentTimeMillis())
+                val duration = (blockZapperCooldownExpiration - System.currentTimeMillis()).milliseconds
```

Comment:
This won't work cause essential

Reply:
IntelliJ auto corrected it without telling me :smile: